src/raftstore/store/peer.rs

```diff
@@ -364,6 +364,13 @@ impl Peer {
     }
 
     pub fn check_peers(&mut self) {
+        if !self.is_leader() {
+            if !self.peer_heartbeats.is_empty() {
```

Comment:
clear method will create a Drain object and depend on the drop method, which is unnecessary when the map is empty.